### PR TITLE
Add cpu notebook

### PR DESCRIPTION
**Pull request by samos123**

### CODE REVIEW

**Review by brandonjbjelland — DISMISSED** [review]
nice, simple, and cheap! 😛

File: examples/notebooks/cpu-notebook.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: substratus.ai/v1
+kind: Notebook
+metadata:
+  name: cpu-notebook
+spec:
+  suspend: false
+  image:
+    git:
+      url: https://github.com/substratusai/notebook-image
```
Comment (nstogner) [inline examples/notebooks/cpu-notebook.yaml:9]: Can we change thing to `notebook-minimal`? I am guessing we will publish a few variants.

**Review by nstogner — CHANGES_REQUESTED** [review]

File: examples/notebooks/cpu-notebook.yaml (hunk 1 above)
Comment (samos123) [reply examples/notebooks/cpu-notebook.yaml:9]: I was going to add a notebook-gpu-image afterwards that's based on the cuda base image.

Why don't I call it `notebook-minimal-cpu`?

Comment (samos123) [reply examples/notebooks/cpu-notebook.yaml]: done

**Review by nstogner — APPROVED** [review]

**Pull request merged**